panther_bringup/launch/bringup.launch

```diff
@@ -9,6 +9,7 @@
   <arg name="launch_lights_scheduler" default="true" />
   <arg name="shutdown_hosts_config_file" default="$(find panther_bringup)/config/shutdown_hosts.yaml" />
   <arg name="disable_manager" default="false" doc="used for testing purposes" />
+  <arg name="kill_on_wrong_hw" default="true" doc="kill ROS when incorrect hardware detected" />
```

Comment:
maybe `exit_on_wrong_hw` instead? so it will be the same as param name of the node